<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RealtyVault</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;logo.png&quot;);
background-position: center;


</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_a_House"/>
    <addaction name="actionModify_a_House"/>
    <addaction name="actionSearch_for_a_House"/>
    <addaction name="actionReport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_Contents"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp_Contents">
   <property name="text">
    <string>Help Contents</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAdd_a_House">
   <property name="text">
    <string>Add a House</string>
   </property>
  </action>
  <action name="actionModify_a_House">
   <property name="text">
    <string>Modify a House</string>
   </property>
  </action>
  <action name="actionSearch_for_a_House">
   <property name="text">
    <string>Search for a House</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report</string>
   </property>
  </action>
  <action name="actionExir">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
